src/test/java/LoginPageTest.java

```diff
@@ -55,8 +54,60 @@ public void loginPage_LoginFormSection_HasIntroductionAndLoginFormCorrectlyDispl
         softAssertions.assertThat(loginPage.isTextForgotYourPasswordDisplayed()).isTrue();
         softAssertions.assertThat(loginPage.getForgotYourPasswordLinkText()).isEqualTo("Forgot Your Password?");
         softAssertions.assertThat(loginPage.getForgotYourPasswordLinkTextColorAsHex()).isEqualTo("#006bb4");
-        softAssertions.assertThat(driver.findElement(By.cssSelector(".secondary .action.remind")).getAttribute("href")).
-                isEqualTo("https://magento.softwaretestingboard.com/customer/account/forgotpassword/");
+        softAssertions.assertThat(loginPage.getForgotYourPasswordLink()).isEqualTo("https://magento.softwaretestingboard.com/customer/account/forgotpassword/");
+        softAssertions.assertAll();
+    }
+
+    @Test
+    public void loginPage_HeaderFooterNavigationDemoCustomer_AreSectionsCorrectlyDisplayed() {
+        //given
+        LoginPage loginPage = new LoginPage(driver);
+
+        //when
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(loginPage.isPageHeaderDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.isPageNavigationBarDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.isPageFooterDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.isNewCustomerContainerDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.isTryDemoCustomerAccessSectionDisplayed()).isTrue();
+        softAssertions.assertAll();
+    }
+
+    @Test
+    public void loginPage_NewCustomerSection_HasTitleAndTextAndButtonCorrectlyDisplayed() {
+        //given
+        LoginPage loginPage = new LoginPage(driver);
+
+        //when
+        SoftAssertions softAssertions = new SoftAssertions();
+        softAssertions.assertThat(loginPage.isNewCustomersTitleDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.getNewCustomersTitleText()).isEqualTo("New Customers");
+        softAssertions.assertThat(loginPage.isNewCustomersMainTextDisplayed()).isTrue();
+        softAssertions.assertThat(loginPage.getNewCustomersMainText()).
+                isEqualTo("Creating an account has many benefits: check out faster, keep more than one address, track orders and more.");
+        softAssertions.assertThat(loginPage.isCreateAccountButtonEnabled()).isTrue();
+        softAssertions.assertThat(loginPage.getCreateAccountButtonText()).isEqualTo("Create an Account");
+        softAssertions.assertThat(loginPage.getCreateAccountButtonLink()).isEqualTo("https://magento.softwaretestingboard.com/customer/account/create/");
+        softAssertions.assertThat(loginPage.getCreateAccountButtonFontColorAsHex()).isEqualTo("#ffffff");
+        softAssertions.assertThat(loginPage.getCreateAccountButtonBackgroundColorAsHex()).isEqualTo("#1979c3");
```

Comment:
These 2 colors strings are used for sure twice in this class. Please extract them to constant class fields and reuse.
And check if there are any other strings that are repeated and extract them as well.